pyradiosky/skymodel.py

```diff
@@ -1807,35 +1993,54 @@ def read_healpix_hdf5(
         )
         assert self.component_type == "healpix"
 
-        if history is None:
-            self.history = self.pyradiosky_version_str
-        elif not uvutils._check_history_version(
-            self.history, self.pyradiosky_version_str
-        ):
-            self.history += self.pyradiosky_version_str
-
         if run_check:
             self.check(
                 check_extra=check_extra, run_check_acceptability=run_check_acceptability
             )
         return
 
     @classmethod
-    def from_votable_catalog(cls, votable_file, *args, **kwargs):
-        """Create a :class:`SkyModel` from a votable catalog.
+    def from_healpix_hdf5(
+        cls,
+        hdf5_filename,
+        run_check=True,
+        check_extra=True,
+        run_check_acceptability=True,
+    ):
+        """
+        Create a new :class:`SkyModel` from a hdf5 healpix file.
+
+        Deprecated. Support for this file format will be removed in version 0.3.0.
+        Use `from_skyh5` to create a new :class:`SkyModel` from our newer skyh5 file type.
```

Comment:
Should we issue this warning when a user calls this function too?

Reply:
I think it will generate a warning because it calls `read_healpix_hdf5` which issues a deprecation warning.